README.md

```diff
@@ -17,6 +17,7 @@ Alternatively, this data set can be accessed by following [this link](https://po
 
 To pass a set of files to this tool, query the desired files and download a GDC Manifest. This can be done by doing any of the following:
 
+- Selecting a project from case facet
```

Comment:
Move this line to around line 15 underneath the "* Data Format: maf" and change it to "Select a Project"